src/components/dismissible/dismissible.js

```diff
@@ -14,26 +14,29 @@ const {Icon, IconType} = Components;
 class DismissibleFloatingButtonComponent extends Component {
   render(props: any) {
     return (
-      <Localizer>
-        <a
-          role="button"
-          ref={el => {
-            if (props.addAccessibleChild) {
-              props.addAccessibleChild(el);
-            }
-          }}
-          tabIndex="0"
-          onClick={() => props.onClose()}
-          onKeyDown={e => {
-            if (e.keyCode === KeyMap.ENTER) {
-              props.onClose();
-            }
-          }}
-          aria-label={<Text id="overlay.close" />}
-          className={'playkit-floating-dismissible playkit-icon playkit-icon-close'}>
-          <Icon type={IconType.Close} />
-        </a>
-      </Localizer>
+      <div id={'playkit-floating-dismissible-container'}>
```

Comment:
Hmmm only if there are multiple players with floating.. i used id because Utils.Dom doesn't have `getElementsByClassName`, i could get it by id.
you think it might cause an issue?